Lib/test/test_compileall.py

```diff
@@ -16,10 +16,14 @@
 import unittest
 
 from unittest import mock, skipUnless
+from concurrent.futures import ProcessPoolExecutor
```

Comment:
Why is this import outside the `try-except`

Reply:
`ProcessPoolExecutor` can always be imported successfully, see e.g. https://github.com/python/cpython/blob/63298930fb531ba2bb4f23bc3b915dbf1e17e9e1/Lib/concurrent/futures/process.py#L572

If the import were within the `try-except` it would still happen successfully, so it doesn't seem to improve anything to do it within the `try-except`. From what I can tell, the `try-except` exists to catch an exception from `_check_system_limits()`.

Since it's peaceful to always import it, and I think it's simpler that way, I moved it out of the `try-except`.

Reply:
Great, thanks for clarifiying